plugin/trino-iceberg/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>log</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
```

Comment:
This used only be a test time dependency but it was telling me that trino hive needs it to be compile time